packages/style-engine/src/styles/padding.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../types';
+import { generateBoxRules } from './utils';
+
+const padding = {
+	name: 'padding',
+	generate: ( style: Style, options: StyleOptions ) => {
+		return generateBoxRules(
+			style,
+			options,
+			[ 'spacing', 'padding' ],
```

Comment:
Could this path value one day be an options value we get from https://github.com/WordPress/gutenberg/blob/4452f092259b93521440aa0a8b01b872cc1350de/packages/blocks/src/api/constants.js#L95?

Just wondering if the style engine needs to know about the block implementation detail, or if we even need a single source of ~vermouth~ truth.

Reply:
Good question! I think having a single source of truth is a good idea, but I imagine that might actually live in this package instead of in the blocks package (if we assume that the style engine is the "owner" of how to process the style object)? I suppose we could either move the constants over to the style engine in a central place, or leave it up to each individual style definition to know only about itself?

Either way, probably a good thing for us to explore in follow-ups!